frontend/spec/helpers/navigation_helper_spec.rb

```diff
@@ -5,28 +5,6 @@ module Spree
     include Spree::CurrencyHelper
     include Spree::LocaleHelper
 
-    describe '#spree_navigation_data' do
```

Comment:
@MatthewKennedy let's keep the test as well :)

Reply:
Sure, and I'll add some for the new methods this afternoon.